tests/test_observation.py

```diff
@@ -1,136 +1,46 @@
 import numpy as np
 from numpy.testing import assert_array_equal, assert_almost_equal
-import astropy.wcs
-
+from functools import partial
 import scarlet
 
 
-def get_airy_wcs():
-    wcs = astropy.wcs.WCS(naxis=2)
-    # Set up an "Airy's zenithal" projection
-    wcs.wcs.crpix = [-234.75, 8.3393]
-    wcs.wcs.cdelt = np.array([-0.066667, 0.066667])
-    wcs.wcs.crval = [0, -90]
-    wcs.wcs.ctype = ["RA---AIR", "DEC--AIR"]
-    wcs.wcs.set_pv([(2, 1, 45.0)])
-    return wcs
-
-
 class TestObservation(object):
     def get_psfs(self, shape, sigmas):
+
+        shape_ = (None, *shape)
         psfs = np.array([
-            scarlet.psf.generate_psf_image(scarlet.psf.gaussian, shape, amplitude=1, sigma=s).image
+            scarlet.PSF(partial(scarlet.psf.gaussian, sigma=s), shape=shape_).image[0]
             for s in sigmas
         ])
 
-        normalized = psfs.copy()
-        normalized /= psfs.sum(axis=(1, 2))[:, None, None]
-
-        return psfs, normalized
-
-    def test_frame(self):
```

Comment:
I think we should test somewhere that our WCS conversion is working properly. I don't see a test that looks for this anymore, so a new module `test_frame` with appropriate units tests should be added.

Reply:
In principle, I agree. However, we have no fixed requirements for this test yet. I suggest we punt until the multi-res stuff comes in, where it's really needed. Will create issue to track.